### ISSUE TITLE
Default values for prediction patterns are translated

### ISSUE THREAD

**Issue by pronguen** [rero/rero-ils#1022]
**Describe the issue**
When I create a new holding for serials, it opens the prediction pattern form with default values, to help me understand the form. But the default values are not correct, or are translated. Anyway, the preview on the right doesn't work (see screenshot).
The preview is displayed only after having entered a call number or a value in any field. Why?

**To Reproduce**
1. Go to professional detailed view of a serial, for instance [this one](https://ils.test.rero.ch/professional/records/documents/detail/1182)
1. Add a holding
1. Complete field "call number" or "location"
1. See preview

**Expected behavior**
There is an example preview on the right that works. Ideally, it works at form opening (we don't have to enter a value in order to see the preview).

**Software version**
ils.test.rero.ch
v.0.9.1

**Screenshots**
![image](https://user-images.githubusercontent.com/8154915/83720806-1d69ac00-a63a-11ea-8cdf-d434f5969503.png)

**Comment by zannkukai** [comment]
numbering and chronology are translated because they are into a `name` fiield into the holding JSON schema. Is it a way to prevent these fields are translated by `schemaform/holdings` API ?

**Comment by iGormilhit** [comment]
@zannkukai The rules for JSON schema string extraction are there: https://github.com/rero/rero-ils/blob/dev/babel.ini#L45

The mechanism is there: https://github.com/rero/rero-ils/blob/dev/rero_ils/modules/babel_extractors.py

**Comment by iGormilhit** [comment]
> The preview is displayed only after having entered a call number or a value in any field. Why?

I think that the default template needs also at least a default value (such as the starting value of the numbering.

**Comment by iGormilhit** [comment]
For the translations of the elements of the prediction template, this is not reproductible on ils.test.rero.ch `v0.12.0rc`.

**Comment by pronguen** [comment]
> For the translations of the elements of the prediction template, this is not reproductible on ils.test.rero.ch `v0.12.0rc`.

Yes, it seems to work fine :+1: (someone did something?!)

> Should we add a default value to get the preview

Currently, after opening the default template, we just need to add a call number so that the preview is display. Ideally, the preview is display directly at opening the editor, without having to enter a call number. But this is not a priority issue.

**Comment by iGormilhit** [comment]
> Yes, it seems to work fine :+1:  (someone did something?!)

I have no idea. At least something happened...

> Currently, after opening the default template, we just need to add a call number

… or a start value for the numbering, which kind of make sens, otherwise I don't know how the preview could work. I agree this issue has a low priority. But we have to decide how to fix this issue. Maybe the record template could help…
